demos/code/Blocks/Chapter 5 - Functions.js

```diff
@@ -765,6 +602,480 @@ Blockly.C['func_return'] = function(block) {
 	return code;
 };
 
+Blockly.Blocks['func_parameters_call'] = {
+	init: function() {
+		this.appendDummyInput()
+			.appendField(new Blockly.FieldDropdown([["","myPtrNone"], ["&","myPtrAdd"], ["*&","myPtrAddPtr"], ["*","myPtrAdd1"], ["**","myPtrAdd2"], ["***","myPtrAdd3"]]), "myPtr")
+			.appendField("Name:")
+			.appendField(new Blockly.FieldVariable("myParam1"), "myParamName");
+
+		this.setInputsInline(false);
+		
+		this.setPreviousStatement(true, null);
+		this.setNextStatement(true, null);
+
+		this.setColour(funcHUE);
+		this.setTooltip("");
+		this.setHelpUrl("");
+	}
+};
+Blockly.C['func_parameters_call'] = function(block) {
+	var variable_myparamname = Blockly.C.variableDB_.getName(block.getFieldValue('myParamName'), Blockly.Variables.NAME_TYPE);
+	var value_valueInput = Blockly.C.valueToCode(block, 'valueInput', Blockly.C.ORDER_ATOMIC);
+	var dropdown_myptr = this.getField('myPtr').getText();
+
+	// TODO: Assemble C into code variable.
+	var code = '';
+
+
+	code += dropdown_myptr + ' ' + variable_myparamname;
+
+	if(value_valueInput.length > 0){
+		code += ', ' + value_valueInput;
+	}
+
+	// TODO: Change ORDER_NONE to the correct strength.
+	return [code, Blockly.C.ORDER_NONE];
+};
+
+Blockly.Blocks['func_call_return'] = {
+	init: function() {
+		this.appendValueInput("valueInput")
+			.setCheck(null)
+			.appendField(new Blockly.FieldVariable("myFunction"), "myFunc");
+
+		this.setInputsInline(false);
+
+		this.setOutput(true, null);
+
+		this.setColour(funcHUE);
+		this.setTooltip("Calls a user defined function.\nInput - Parameters defined");
+		this.setHelpUrl("https://www.tutorialspoint.com/cplusplus/cpp_functions.htm");
+		
+		this.setMutator(new Blockly.Mutator(['func_var_init_literal', 'func_parameters_call']));
+
+	},
+
+	mutationToDom: function(){
+		var container = document.createElement('mutation');
+		return container;
+	},
+
+	domToMutation: function(xmlElement){
+
+	},
+
+	decompose: function(workspace){
+		var containerBlock = workspace.newBlock(['function_mutator']);
+		containerBlock.initSvg();
+		return containerBlock;
+	},
+
+	compose: function(containerBlock){
+		
+	}
+};
+Blockly.C['func_call_return'] = function(block) {
```

Comment:
This is just a rename of the block. The reference in index.html has been updated as well.